diff --git a/node_modules/cypress/types/cypress-expect.d.ts b/node_modules/cypress/types/cypress-expect.d.ts
index 74f71a5..28dd8f2 100644
--- a/node_modules/cypress/types/cypress-expect.d.ts
+++ b/node_modules/cypress/types/cypress-expect.d.ts
@@ -1,3 +1,3 @@
 // Cypress adds chai expect and assert to global
-declare const expect: Chai.ExpectStatic
-declare const assert: Chai.AssertStatic
+// declare const expect: Chai.ExpectStatic
+// declare const assert: Chai.AssertStatic
